README.md

```diff
@@ -8,8 +8,7 @@ This website was created by @avasconcelos114 and @josephk96 for the 2019 Hacktob
 
 ## Code Style
```

Comment:
Thanks for the PR and for letting me know about the changes in the original documentation!

I'd say your proposal to the documentation changes is pretty good :)

My only feedback is that it might be more clear to rename this section to something like `Code Block Styles`

Reply:
@avasconcelos114 Great feedback! Updated